xpdacq/tests/test_beamtimeSetup.py

```diff
@@ -94,9 +94,9 @@ def test_start_beamtime(self):
         self.assertEqual(self.experimenters, self.bt.get('bt_experimenters'))
         self.assertEqual(self.wavelength, self.bt.get('bt_wavelength'))
         self.assertEqual(os.getcwd(), self.home_dir)
-        # FIXME
-        # strtScnLst = ['bt_bt.yml','ex_l-user.yml','sa_l-user.yml','sp_ct_0.1.yml','sp_ct_0.5.yml','sp_ct_1.yml','sp_ct_5.yml','sp_ct_10.yml','sp_ct_30.yml']
-        # self.assertEqual(newobjlist,strtScnLst)
+        # test prepoluate ScanPlan
+        self.assertEqual(len(self.bt.scanplans), 3)
+        self.assertEqual(self.bt.scanplans[0]['sp_args'], (5,))
```

Comment:
this test is ok, but it is not testing much of the functionality is it?

Also, I think we wanted  5s, "min", 1s,  10s, 30s, 60s in the prepopulated list, something like that (with 5s in the first place)?